### ISSUE TITLE
readthedocs_ext crashes with 1.6b2

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#3683]
### Problem
- readthedocs_ext crashes with 1.6b2

#### Error logs / results
```
Running Sphinx v1.6+
making output directory...
loading translations [en]... done
Generating grammar tables from /home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/pycode/Grammar-py2.txt
/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/websupport/__init__.py:26: UserWarning: Since Sphinx-1.6, sphinx.websupport module is now separated to sphinxcontrib-webuspport package. Please add it into your dependency list.
  warnings.warn('Since Sphinx-1.6, sphinx.websupport module is now separated to '

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/application.py", line 195, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/application.py", line 455, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170429-py2.7.egg/sphinx/registry.py", line 199, in load_extension
    raise ExtensionError(_('Could not import extension %s') % extname, err)
ExtensionError: Could not import extension readthedocs_ext.readthedocs (exception: No module named sphinxcontrib.websupport.storage)

Extension error:
Could not import extension readthedocs_ext.readthedocs (exception: No module named sphinxcontrib.websupport.storage)
```
http://readthedocs.org/projects/sphinx/builds/5356125/

**Comment by tk0miya** [comment]
In #3681, I made websupport module optional. But it breaks compatibility with 1.5 series.

**Comment by tk0miya** [comment]
I added `sphinxcontrib-websupport` to dependency list again, but it still failed
```
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170503-py2.7.egg/sphinx/themes/basic/../basic/layout.html", line 109, in template
    <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/jinja2/sandbox.py", line 427, in call
    return __context.call(__obj, *args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170503-py2.7.egg/sphinx/builders/html.py", line 906, in pathto
    uri = relative_uri(baseuri, otheruri) or '#'
  File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx/envs/1.6-release/lib/python2.7/site-packages/Sphinx-1.6.dev20170503-py2.7.egg/sphinx/util/osutil.py", line 61, in relative_uri
    if to.startswith(SEP):
UndefinedError: 'str object' has no attribute 'filename'
```

**Comment by tk0miya** [comment]
It is caused that `readthedocs_ext` extension calls `builder.css_files.insert()`. We does not handle it.
Finally, I fixed it on 1.6-release branch.
